test/mock/rtps/ReaderLocator/rtps/writer/ReaderLocator.hpp

```diff
@@ -258,4 +256,4 @@ class ReaderLocator : public RTPSMessageSenderInterface
 } /* namespace eprosima */
 
 #endif // ifndef DOXYGEN_SHOULD_SKIP_THIS_PUBLIC
-#endif /* _FASTDDS_RTPS_READERLOCATOR_H_*/
+#endif /* RTPS_WRITER__READERLOCATOR_HPP*/
```

Comment:
My OCD prefers double slash and double space here:

```suggestion
#endif  // RTPS_WRITER__READERLOCATOR_HPP
```